src/credentials/FabricTable.h

```diff
@@ -411,6 +411,13 @@ class DLL_EXPORT FabricTable
 {
 public:
     FabricTable() { Reset(); }
+    ~FabricTable()
+    {
+        if (mlistenerOwneship)
+        {
+            chip::Platform::MemoryFree(mFabricListener);
```

Comment:
Shouldn't this be a platform delete?  This is not running destructors....

Reply:
Fixed